### DISCUSSION TITLE
Tasmota to HAA

### DISCUSSION THREAD

**Discussion by PhaseAngle** [RavenSystem/esp-homekit-devices#1899] [General]
On the Installation page under Third-Party Installation Tools
an extra note for Tasmota to HAA should be added

Newer versions of Tasmota may first require you to flash with tasmota-lite.bin as 1MB devices will not have 404kB of free space

**Comment by RavenSystem** [comment]
You must use [tasmota2haa.bin.gz](https://github.com/RavenSystem/haa/releases/latest/download/tasmota2haa.bin.gz) (281 KB) instead `fullhaaboot.bin` file.

**Reply by PhaseAngle** [reply]
Sorry, I forgot about Apples auto clean up of downloaded compressed files.

**Reply by RavenSystem** [reply]
You can disable it in Safari -> Preferences -> General.
